test/e2e/framework/util.go

```diff
@@ -3410,7 +3410,7 @@ func IssueSSHCommand(cmd, provider string, node *v1.Node) error {
 }
 
 // NewHostExecPodSpec returns the pod spec of hostexec pod
-func NewHostExecPodSpec(ns, name string) *v1.Pod {
+func NewHostExecPodSpec(ns, name string, hostNetwork bool) *v1.Pod {
```

Comment:
Might want to rename this to `NewExecPodSpec`, since it is no longer strictly host networked.

Reply:
Done